scania-digital-design-system/sdds
# feat(side menu): sticky collapse button in react demo

**Describe pull-request**  
sticky collapse button on side menu in React demo 

**Solving issue**  
Fixes: [AB#1372](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1372)

**How to test**  
1. Go to React demo
2. Check in side menu collapse button
3. The button should be always shown on the bottom of screen and able to be focused when you tab around

**Additional context**  
I added few classes to our component package(components/src/patterns/side-menu/side-menu.scss)
and kept the same ones in React demo locally(demo/react/src/index.css) for now to see the result before we release the package. I'm planning to clean up the duplicate in React once we release it.
🤔  but please let me know if it's too much? perhaps it's better to make a PR for the package update first and update demo later?
